pkg/daemon/cleanup/cleanup.go

```diff
@@ -104,3 +106,10 @@ func removeServices(svcMgr manager.Manager, services []servicescm.Service) error
 	}
 	return nil
 }
+
+// killContainerdContainers makes a best effort to stop all processes with the name containerd-shim-runhcs-v1
+func killContainerdContainers() {
```

Comment:
I suggest a gentler name like `deleteContainerdContainers()` or just `cleanupContainers()`. You are not deleting containers per se in some sense.